README.md

```diff
@@ -1,4 +1,33 @@
 # Federal Grant Reporting Project
+Improving the experience of federal grant reporting.
+
+# Single audit resolution
+Enabling simpler, faster, easier, better resolution of single audit findings by
+agencies and grantees alike.
+
+## Local installation
+
+This is a Django app designed to run on Python 3.7.3.
+[pipenv](https://pipenv.readthedocs.io) is recommended for creating a virtual
+environment and managing dependencies. If you don't already have `pipenv`
+installed:
+
+1. Confirm that your `pip` uses Python 3, not Python 2, then
```

Comment:
Small thing (not a blocker), but if you know off the top of your head — what would someone type on the command line to verify that their `pip` is using Python 3? 

Emoji-reply with 🚀 if you'd rather skip this comment or address it later and bias towards 🚢-ing instead.

Reply:
:) It occurred to me to include that, but I was biasing toward 🚢-ing. I'll go ahead and add it.

Reply:
Added in https://github.com/18F/federal-grant-reporting/pull/83/commits/644b491e8f41a802b39e33404bd45db81d463a27. Thanks!